pkgs/desktops/gnome-3/3.16/default.nix

```diff
@@ -176,6 +176,7 @@ let
   libcroco = callPackage ./core/libcroco {};
 
   libgee = callPackage ./core/libgee { };
+  libgee_1 = callPackage ./core/libgee/libgee-1.nix { };
```

Comment:
That is because this version of libgee is also known as libgee-1.0. See https://wiki.gnome.org/Projects/Libgee in the section 'Versions'.